engine/src/main/java/org/kakara/engine/item/Texture.java

```diff
@@ -79,7 +79,10 @@ public Texture(Resource resource, Scene currentScene) {
                 decodedImage = stbi_load_from_memory(resource.getByteBuffer(), w, h, avChannels, 4);
             }else{
                 decodedImage = stbi_load(resource.getPath(), w, h, avChannels, 4);
-
+            }
+            if(decodedImage==null) {
+                System.out.println("Failure");
```

Comment:
Change this to a log message that specifies where this is.

Reply:
IDK, I think Failure gets the point across. 

I will change it.

Reply:
What failed though? We should be more specific.